pyaedt/icepak.py

```diff
@@ -6072,6 +6072,7 @@ def create_geometry_from_csv(self, csv_file, header_line=2, column_mapping=None,
             The default value is 2.
         column_mapping : dict, optional
             Map of the csv columns names to the accepted column names.
```

Comment:
```suggestion
            The default is ``2``.
        column_mapping : dict, optional
            Map of the CSV columns names to the accepted column names.
```
Minor edits to follow our style guidelines.